ereuse_devicehub/inventory/views.py

```diff
@@ -0,0 +1,105 @@
+import flask
+from flask.views import View
+from flask import Blueprint, url_for
+from flask_login import login_required, current_user
+
+from ereuse_devicehub.resources.lot.models import Lot
+from ereuse_devicehub.resources.device.models import Device
+from ereuse_devicehub.inventory.forms import LotDeviceForm, LotForm
+
+devices = Blueprint('inventory.devices', __name__, url_prefix='/inventory')
+
+
+class DeviceListView(View):
+    decorators = [login_required]
+    template_name = 'inventory/device_list.html'
+
+    def dispatch_request(self, id=None):
```

Comment:
Does `id` parameter reference to lot id? Then better rename it to something like `lot_id`

Reply:
done